cli/cmd/externalservice/create.go

```diff
@@ -38,7 +38,9 @@ func (c *Create) Run(ctx *clicontext.CLIContext) error {
 				externalService.Spec.TargetServiceNamespace = ref.Namespace
 			} else {
 				externalService.Spec.TargetApp = ref.App
-				externalService.Spec.TargetVersion = ref.Version
+				if strings.Contains(name, "@") {
```

Comment:
I don't understand this, `ParseID` returns the version, what does `@` sign signify ?

Reply:
if format is `rio external create ext app`  then externalservice points to `app`

if format is `rio external create ext app@version` then externalservice points to `app` and `version`.

Reply:
Makes sense. A helper method for app vs app/version might be nice and more obvious at first glance. But fine for now.